cmd/openshift-install/testdata/agent/configimage/manifests/default_manifests.txt

```diff
@@ -0,0 +1,145 @@
+# Verify that the basic manifests are properly generated within the /etc/assisted/manifests folder
+
+exec openshift-install agent create config-image --dir $WORK
+
+exists $WORK/agentconfig.noarch.iso
+
+isocmp agentconfig.noarch.iso /etc/assisted/manifests/agent-cluster-install.yaml expected/agent-cluster-install.yaml
```

Comment:
Would also be work checking that /etc/assisted/manifests/infraenv.yaml is not included in the config.iso.

Reply:
Will add this as a follow-on as it looks like will be able to reuse ignitionContains() in https://github.com/openshift/installer/pull/7186 when that merges.